public/js/views/animations.coffee

```diff
@@ -0,0 +1,100 @@
+BaseView = require "./baseView.coffee"
+
+module.exports = BaseView.extend
+
+  #TODO: Animations should be connected to model
+  #temporary animations generator
+
+  generateAnimation: (canvas, self) ->
+    #number between 1 and 3
+    num = Math.floor((Math.random() * 3) + 1)
+
+    switch num
+      when 1
+        @rectflash(canvas, self)
+      when 2
+        @circleslide(canvas, self)
+      when 3
+        @linerandom(canvas, self)
+    return
+
```

Comment:
If you init `animations = [@rectflash, @circleslide, @linerandom]` and `num = parseInt((Math.random() * 10) % 3)`, you can do `animations[num](canvas, self)` instead of the switch-case?